src/map/script_constants.hpp

```diff
@@ -9624,6 +9624,10 @@
 	export_constant(HAT_EF_C_ROS2023_CAPE_1);
```

Comment:
```suggestion
	export_constant(HAT_EF_C_ROS2023_CAPE_1);
	export_constant(HAT_EF_BLACK_THUNDER_);
```

Reply:
Oh yes, i have not checked this, thanks!